### ISSUE TITLE
Variable editing panel does not expand on keyboard focus

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1095]
Repro Steps: 
Launch 'Calculator'.
Home screen will be displayed.​
Navigate through Menu using keyboard and select 'Graphing'. 'Graphing' Calculator opens.​
Navigate to the  expand/collapsed icon below sliders.
Observe the issue. 

Actual Result:
"Expand/Collapse" icon button which is present below slider is not accessible via keyboard. 

Expected Result:
Expand/collapsed icon button which is present below slider should be accessible via keyboard.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
